### ISSUE TITLE
ActNorm is incorrectly applied

### ISSUE THREAD

**Issue by mjack3** [vislearn/FrEIA#172]
In the code the inverse is applied in the forward step and the forward step in the reverse.

**Comment by fdraxler** [comment]
Thanks for raising this issue! Can you maybe produce a minimal code example reproducing the issue so that we can have a closer look? Thanks!

**Comment by mjack3** [comment]
You can reproduce the error by ingesting a simple tensor of shape batch x channnel x height x width.

For example:

x = torch.rand(32,128,28,28)

And then ingest it to a INN using the sequinn API

**Comment by ju-w** [comment]
I fail to see the problem.

```python
def build():
    nodes = [     Ff.InputNode(3,32,32) ]
    nodes.append( Ff.Node(nodes[-1], Fm.ActNorm) )
    nodes.append( Ff.OutputNode(nodes[-1]))
    inn = Ff.GraphINN(nodes)
    return inn
inn = build()
x, jac_det = inn(torch.rand(190,3,32,32))
print(jac_det.shape)
```
As expected
```
torch.Size([190])
```
Of course it is 190 times the same number, as `jac_det` is only determined by the scaling inside ActNorm.

**Comment by mjack3** [comment]
You are right. I have checked it in the main and 0.2 versions. Both work fine. Sorry for the false alarm. This issue can be closed :)
